src/Microsoft.Identity.Web.TokenAcquisition/MergedOptions.cs

```diff
@@ -213,6 +213,9 @@ internal static void UpdateMergedOptionsFromMicrosoftIdentityOptions(MicrosoftId
             mergedOptions.StringDataFormat ??= microsoftIdentityOptions.StringDataFormat;
 #if NET8_0_OR_GREATER
             mergedOptions.TimeProvider = microsoftIdentityOptions.TimeProvider;
+#endif
+#if NET9_0_OR_GREATER
+            mergedOptions.AdditionalAuthorizationParameters = microsoftIdentityOptions.AdditionalAuthorizationParameters;
```

Comment:
nit: I think the properties should be in alphabetical order (mostly)?

Reply:
no. This is based on the order of the properties in the class itself.